robottelo/helpers.py

```diff
@@ -750,3 +749,190 @@ def host_provisioning_check(ip_addr):
         raise ProvisioningCheckError(
             'Failed to ping virtual machine Error:{0}'.format(
                 result.stdout))
+
+
+# --- Issue based Pytest markers ---
+
+def is_open(issue, data=None):
+    """Check if specific issue is open.
+
+    Issue must be prefixed by its handler e.g:
+
+    Bugzilla: BZ:123456
+
+    Arguments:
+        issue {str} -- A string containing handler + number e.g: BZ:123465
+        data {dict} -- Issue data indexed by <handler>:<number> or None
+    """
+    # Handlers can be extended to support different issue trackers.
+    handlers = {'BZ': bugzilla.is_open_bz}
+    supported_handlers = tuple(handlers.keys())
+
+    if str(issue).startswith(supported_handlers):
+        handler_code, _, _ = str(issue).partition(":")
```

Comment:
Technically, this branch will execute for issue=`BZ1234`. `handler_code` will then be set to `issue` variable value, eventually throwing Key Error in last `return`.

Possibly you want slightly different definition of supported_handlers:
```python
supported_handlers = tuple(f"{handler}:" for handler in handlers.keys())
```